tracer/resources/args_parser.py

```diff
@@ -16,7 +16,7 @@
 # 02110-1301, USA.
 #
 
-import argparse
+import argparse, argcomplete
```

Comment:
@TingPing that makes sense! :+1: But what distros doesn't have argcomplete packaged?